apps/api/lib/validations/shared/queryUserEmail.ts

```diff
@@ -10,7 +10,7 @@ export const schemaQueryUserEmail = baseApiParams.extend({
 });
 
 export const schemaQuerySingleOrMultipleUserEmails = z.object({
-  email: z.union([z.string().email(), z.array(z.string().email())]).optional(),
+  email: z.union([z.string().email(), z.array(z.string().email())]),
```

Comment:
If we remove optional here, would the endpoint still work if no email is provided?  🤔

Reply:
Then req.email is falsy:
<img width="747" alt="image" src="https://github.com/calcom/cal.com/assets/1046695/03f58cae-5e24-49f7-bbad-a185ca4c12bc">

Reply:
Oh yes, good call!